units/periodic_builds.go

```diff
@@ -193,7 +194,7 @@ func (j *periodicBuildJob) addVersion(ctx context.Context, definition model.Peri
 		PeriodicBuildID: definition.ID,
 		Alias:           definition.Alias,
 		Revision: model.Revision{
-			Revision: mostRecentVersion.Revision,
+			Revision: mostRecentRevision,
```

Comment:
Since it's possible for `mostRecentRevision` to be empty here, should we retrieve the commit hash from the master branch since that is what `GetGithubFile` would do? Otherwise I think the created version will have an empty revision

Reply:
Shouldn't be relevant now that we return if mostRecentRevision is empty